packages/edit-widgets/src/store/reducer.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { combineReducers } from '@wordpress/data';
+
+/**
+ * Reducer returning the editor setting.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function settings( state = {}, action ) {
+	switch ( action.type ) {
+		case 'UPDATE_WIDGET_EDITOR_SETTINGS':
+			return {
+				...state,
+				...action.settings,
+			};
+	}
```

Comment:
Do we really need a store for this, if feels like for now we can just pass these media options as props to the block editors.

Reply:
We are going to need an edit-post store anyway, so I took the chance to add the store here to make the PR that was adding the store smaller.
Provided we will have a store, adding the settings to the store makes this implementation consistent with what we do in other modules.

Reply:
> Provided we will have a store, adding the settings to the store makes this implementation consistent with what we do in other modules.

To be clear I'm not sure it's a good idea in the other modules (editor) too, it's just because we built that module first that it's there. I think it will simplify things a lot if we avoid it. 

I'm not against introducing it later if we feel the need but I'm really unsure about it. It feels like we're just going to pass these settings down.

Reply:
(And yea, we'll need a store that's true, it's more the settings in the store that bothers me)